en/cloud_development_setup/instructions.md

```diff
@@ -0,0 +1,24 @@
+You can [skip right over this section](http://tutorial.djangogirls.org/en/installation/#install-python) 
+if you're not using RunCode Cloud Development environment. If you are, your installation experience will be a little different. 
```

Comment:
Move "If you are" to a new line

Reply:
And add in - "If you are installing the RunCode Cloud Development environment, your installation expereince will be a littel different. You do not need to insall anything locally, you just need to create three (free) online accounts.